applications/working_with_helm_charts/understanding-helm.adoc

```diff
@@ -0,0 +1,36 @@
+[id="understanding-helm"]
+= Understanding Helm
+include::modules/common-attributes.adoc[]
+:context: understanding-helm
+
+toc::[]
+
+Helm is a software package manager that simplifies deployment of applications and services to {product-title} clusters.
+
+Helm uses a packaging format called _charts_.
+A Helm chart is a collection of files that describes the {product-title} resources.
+
+A running instance of the chart in a cluster is called a _release_. A new release is created every time a chart is installed on the cluster.
+
+Each time a chart is installed, or a release is upgraded or rolled back, an incremental revision is created.
+
+
+== Key features
+
+Helm provides the ability to:
+
+* Search through a large collection of charts stored in the chart repository.
+* Modify existing charts.
+* Create your own charts with {product-title} or Kubernetes resources.
+* Package and share your applications as charts.
+
+== Red Hat Certification of Helm charts for OpenShift
+
+You can verify and certify your Helm charts by Red Hat for all the components you will be deploying using the Helm charts on the Red Hat {product-title}. Charts go through an automated Red Hat OpenShift certification workflow that guarantees security compliance as well as best integration and experience with the platform. Certification assures the integrity of the chart and ensures that the Helm chart works seamlessly on Red Hat OpenShift clusters.
```

Comment:
This does not sound well to me, I would simplify it like this: You can choose to certify your Helm Charts on Red Hat OpenShift. 
I think we can even remove that first sentence.

Reply:
@dperaza4dustbit As discussed, changing the content from _You can verify and certify_ to _You can  choose to verify and certify_.